perfkitbenchmarker/providers/aws/aws_disk.py

```diff
@@ -33,12 +33,49 @@
 VOLUME_EXISTS_STATUSES = frozenset(['creating', 'available', 'in-use', 'error'])
 VOLUME_DELETED_STATUSES = frozenset(['deleting', 'deleted'])
 VOLUME_KNOWN_STATUSES = VOLUME_EXISTS_STATUSES | VOLUME_DELETED_STATUSES
+
+STANDARD = 'standard'
+GP2 = 'gp2'
+IO1 = 'io1'
+
 DISK_TYPE = {
-    disk.STANDARD: 'standard',
-    disk.REMOTE_SSD: 'gp2',
-    disk.PIOPS: 'io1'
+    # no need to translate 'standard' because the old and new names
```

Comment:
The DISK_TYPE map was used for that in the Azure provider (it will not be after this PR is applied), but in AWS this map has only been used to translate disk type names.

In the new system, the DISK_METADATA map is the one that checks the validity of the disk type, because you have to look up the disk name in the DISK_METADATA map in order to create a disk spec. I wasn't really trying to set up a validity check, though, so there are cases you can get a bad disk type past it. I thought it would make more sense to leave the checking up to the provider CLIs. If you want, I can implement a real validity check for disk type names.